src/Microsoft.AspNetCore.SignalR/IHubClients.cs

```diff
@@ -13,6 +13,8 @@ public interface IHubClients
 
         IClientProxy Client(string connectionId);
 
+        IClientProxy Others(string connectionId);
```

Comment:
This is the reason the other SignalR had different interfaces for what is essentially IHubClients. There were 2 of them:

IHubConnectionContext -
 https://github.com/SignalR/SignalR/blob/b73cbabe69bc48c4cb8d768148e157712a264ce9/src/Microsoft.AspNet.SignalR.Core/Hubs/IHubConnectionContext.cs

IHubCallerConnectionContext - https://github.com/SignalR/SignalR/blob/b73cbabe69bc48c4cb8d768148e157712a264ce9/src/Microsoft.AspNet.SignalR.Core/Hubs/IHubCallerConnectionContext.cs

Reply:
Yeah, I was looking at old SignalR for reference. I don't think we can expose `Others` the same way with our current implementation. Should we move towards that model?